tests/development-setup.yml

```diff
@@ -0,0 +1,148 @@
+            - "deb-src http://ppa.launchpad.net/ubuntu-lxc/lxd-stable/ubuntu {{ansible_distribution_release}} main"
+          tags: deluge
+
+        - name: Install the Libreswan dependencies that are required for compilation
+          apt:
+            name: "{{ item }}"
+          with_items: "{{ libreswan_compilation_dependencies_trusty }}"
+          when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'trusty'
+
+      #Since LXC loads kernal modules from the host, we need to ensure libreswan is compiled and installed
+    - name: Include Libreswan vars
+      include_vars: "../playbooks/roles/l2tp-ipsec/vars/main.yml"
+
+    - name: Install the Libreswan dependencies that are required for compilation
+      apt:
+        name: "{{ item }}"
+      with_items: "{{ libreswan_compilation_dependencies_trusty }}"
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'xenial'
+
+    - name: Install xmlto (without the extraordinarily large number of packages it normally recommends)
+      apt:
+        name: xmlto
+        install_recommends: no
+
+    - name: Retrieve the Libreswan source code
+      get_url:
+        url: https://download.libreswan.org/{{ libreswan_source_filename }}
+        dest: "{{ libreswan_source_location }}"
+        checksum: "{{ libreswan_checksum }}"
+        owner: root
+        group: root
+        mode: 0644
+      register: libreswan_get_result
+      until: libreswan_get_result|success
+      retries: 3
+      delay: 5
+
+    - name: Extract the Libreswan source code
+      unarchive:
+        copy: no
+        src: "{{ libreswan_source_location }}"
+        dest: /usr/local/src
+        owner: root
+        group: root
+        creates: "{{ libreswan_compilation_directory }}/README"
+
+    - name: Compile and install Libreswan (this will take a while)
+      shell: make -j {{ ansible_processor_cores }} programs && make install
+      args:
+        chdir: "{{ libreswan_compilation_directory }}"
+
+    - name: Start ipsec
+      service:
+        name: ipsec
+        state: started
+        enabled: yes
+      register: ipsec_start
+
+    - name: Pause for a few seconds to let ipsec start
+      pause:
+        seconds: 5
+
+    - name: Debug ipsec start
+      debug:
+        msg: "{{ ipsec_start }}"
+
+    - name: Ipsec verify
+      command: ipsec verify
+      ignore_errors: yes
+
+    - name: Install lxd
+      apt:
+        name: lxd
+        state: present
+        update_cache: yes
+
+    - name: lxd new group
+      shell: newgrp lxd
+
+    - name: lxd init config
+      shell: lxd init --auto --storage-backend dir
+      ignore_errors: yes
+
+    - name: lxd create network
+      shell: lxc network create testbr0
+      ignore_errors: yes
+
+    - name: lxd attach network to default profile
+      shell: lxc network attach-profile testbr0 default eth0
+      ignore_errors: yes
```

Comment:
if you were to run the development-setup.yml on a test machine more than once, this would error.. I figured it was ok to ignore it.